scripts/models/gradient-token/gradient-token.js

```diff
@@ -0,0 +1,204 @@
+const Token = require("../token");
+
+/**
+ * Token used to represent a gradient token collection.
+ */
+class GradientToken extends Token {
+  /**
+   * Construct a new ColorToken.
+   *
+   * @param {Token.Config} config GradientToken configuration Object.
+   */
+  constructor(config) {
+    super({ category: Token.CONSTANTS.CATEGORIES.COLOR, ...config });
+  }
+
+  /**
+   * Merge a GradientToken Object's data into this GradientToken Object's data.
+   *
+   * @param {DTO} dto
+   * @param {GradientToken.Data} dto.data Data to be merged into this Token Object's data.
+   * @returns {This} This GradientToken.
+   */
+  mergeData({ data }) {
+    this.data = { gradation: GradientToken.mergeGradations({ destination: this.data.gradation, source: data.gradation }) };
+
+    return this;
+  }
+
+  /**
+   * Normalize the Data of this GradientToken.
+   *
+   * @returns {This} This GradientToken.
+   */
+  normalizeData() {
+    this.data = { gradation: GradientToken.normalizeGradations({ format: this.format, gradations: this.data.gradation }) };
+
+    return this;
+  }
+
+  /**
+   * Merge GradientToken Gradations into an existing GradientToken's Gradations.
+   *
+   * @param {DTO} dto
+   * @param {GradientToken.Gradations} dto.source GradientToken Gradations to merge into the destination GradientToken Gradations.
+   * @param {GradientToken.Gradations} [dto.destination] GradientToken Gradations to be receiving a merge.
+   * @returns {GradientToken.Gradations} Merged GradientToken Gradations.
+   */
+  static mergeGradations({ destination = {}, source }) {
+    return Object.entries(source).reduce(
+      (final, [gradation, themes]) => ({
+        ...final,
+        [gradation]: GradientToken.mergeThemes({ destination: destination[gradation], source: themes }),
+      }),
+      { ...destination }
+    );
+  }
+
+  /**
+   * Merge GradientToken Themes into an existing GradientToken's Themes.
```

Comment:
I guess ColorToken's Grades is the equivalent of Themes in Gradation. Are 'themes' our momentum themes (dark, light), or what do you mean by them?

Reply:
I see why you did this... is it because the momentum-abstract gradation tokens come with a theme layer (indigo, webex, etc...)?

Reply:
That is correct, they are not, grades are typically numeric: 0-100, while gradation themes are light/dark.